src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/PackageBuilder.cs

```diff
@@ -25,14 +25,26 @@ public class PackageBuilder : IPackageMetadata
         private static readonly Uri DefaultUri = new Uri("http://defaultcontainer/");
         internal const string ManifestRelationType = "manifest";
         private readonly bool _includeEmptyDirectories;
+        private readonly bool _deterministic;
+        private static readonly DateTime ZipFormatMinDate = new DateTime(1980, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Not that it changes anything, but I went ahead and checked if there were any other considerations why we picked a date in 1985 for npm's version of this constant date, and it turns out it really was -only- because it was after 1980 + the Back to the Future reference. So yes, this date should be fine for any platforms that npm ever had to run into, and no one really had complaints besides the weirdness of this date being chosen. I assume this'll be communicated to users somewhere but I didn't see docs on the purpose of this in the PR?

Reply:
The original changes by Cameron had this set to 1970, for which there were ZipArchiveEntry-related exceptions (https://github.com/NuGet/NuGet.Client/pull/2775#issuecomment-481353758) that were resolved by rolling this forward to honor limits there, I believe.  It's possible that's where the date in this PR comes from.

Reply:
Yeah, totally. I was just checking if there were any other issues besides the Zip min date that we'd run into, and I wanted to express that yes, this date is in fact ok in my experience :)

Reply:
Yeah the sole reason is because it's the min date the format allows. 

Regarding docs, they go in https://github.com/NuGet/docs.microsoft.com-nuget and usually are done later at release time. 

Created a tracking issue for myself so I don't miss that :) https://github.com/NuGet/Home/issues/8437

Not sure if the date stuff necessarily ends up in the docs, but I can write up a quick spec.